os_net_config/objects.py

```diff
@@ -1429,18 +1465,32 @@ def __init__(self, name, use_dhcp=False, use_dhcpv6=False, addresses=None,
     @staticmethod
     def update_vf_config(iface, driver=None):
         if iface.trust is None:
-            logger.info("Trust is not set for VF %s:%d, defaulting to on"
-                        % (iface.device, iface.vfid))
+            logger.info(
+                "%s-%d: Trust is not set, defaulting to on",
+                iface.device,
+                iface.vfid,
+            )
             iface.trust = "on"
         if iface.spoofcheck is None:
-            logger.info("Spoofcheck is not set for VF %s:%d, defaulting to off"
-                        % (iface.device, iface.vfid))
+            logger.info(
+                "%s-%d: Spoofcheck is not set, defaulting to off",
+                iface.device,
+                iface.vfid,
+            )
             iface.spoofcheck = "off"
         if iface.promisc is not None:
-            logger.warning("Promisc can't be changed for ovs_dpdk_port")
+            logger.warning(
+                "%s-%d: Promisc can't be changed for ovs_dpdk_port",
```

Comment:
Like the above examples, can this be re-phrased as "Promisc can't be set on ovs_dpdk_port, defaulting to None" ?

Reply:
Its different - Here the **warning** is raised when promisc is set.

Reply:
But we are defaulting to None - not returning